<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NvidiaDextrSegTool3DGUI</class>
 <widget class="QWidget" name="NvidiaDextrSegTool3DGUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>410</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="hModelsLabel">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Models</string>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="modelsLabel">
     <property name="text">
      <string>&lt;p&gt;Change server URI in Nvidia AIAA preferences (Ctrl+P).&lt;/p&gt;&lt;p&gt;&lt;a href='http://0.0.0.0:5600/v1/models'&gt;Click here&lt;/a&gt; to see ALL available Models&lt;/p&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line1">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="hSegmentationLabel">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Auto Segmentation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="autoSegmentationBtn">
     <property name="text">
      <string>Run Segmentation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="sendFeedBackCkBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Use Feedback In Annotation</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="hAnnotaitonLabel">
     <property name="font">
      <font>
       <family>DejaVu Sans</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Annotation</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="addPointsLabel">
     <property name="text">
      <string>(Shift + Left click) to add points</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="openExternalLinks">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="clearPointsBtn">
     <property name="text">
      <string>Clear Points</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="confirmPointsBtn">
     <property name="text">
      <string>Confirm Points</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
